3-UsingAPIs/Week2/homework/ex2-pokemonApp/index.js

```diff
@@ -18,18 +18,47 @@ Complete the four functions provided in the starter `index.js` file:
 Try and avoid using global variables. Instead, use function parameters and 
 return values to pass data back and forth.
 ------------------------------------------------------------------------------*/
-function fetchData(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+async function fetchData(url) {
+  const data = await fetch(url);
```

Comment:
You should also check for HTTP errors:

```suggestion
  const res = await fetch(url);
  if (!res.ok) {
    throw new Error(...);
  }
```